src/utils/uniqueId.ts

```diff
@@ -1,3 +1,5 @@
+// Note: uniqueId may be unsafe in SSR contexts if it is used create DOM IDs or otherwise cause a hydration warning. Use useSSRSafeId instead.
+
 let idSeed = 10000
 export function uniqueId(): string {
```

Comment:
Is `uniqueId` still useful if we have `useSSRSafeId`?

Reply:
There's one place where it's still used — the `useFocusZone` behavior. I believe it's safe because it's only called downstream of a user interaction. As far as I can tell, we'd need to do a decent refactoring or just require that IDs are already in place to get rid of it.